src/acceptance-test/java/net/consensys/orion/acceptance/send/receive/privacyGroup/tx/PushToPrivacyGroupTest.java

```diff
@@ -0,0 +1,172 @@
+    }
+
+    Config firstNodeConfig = NodeUtils.nodeConfig(
+        tempDir,
+        0,
+        "127.0.0.1",
+        0,
+        "127.0.0.1",
+        "node1",
+        joinPathsAsTomlListEntry(key1pub),
+        joinPathsAsTomlListEntry(key1key),
+        "off",
+        "tofu",
+        "tofu",
+        "leveldb:database/node1");
+
+    Config secondNodeConfig = NodeUtils.nodeConfig(
+        tempDir,
+        0,
+        "127.0.0.1",
+        0,
+        "127.0.0.1",
+        "node2",
+        joinPathsAsTomlListEntry(key2pub),
+        joinPathsAsTomlListEntry(key2key),
+        "off",
+        "tofu",
+        "tofu",
+        "sql:" + jdbcUrl);
+
+    firstOrionLauncher = NodeUtils.startOrion(firstNodeConfig);
+    firstHttpClient = vertx.createHttpClient();
+    Orion secondOrionLauncher = NodeUtils.startOrion(secondNodeConfig);
+
+    Box.PublicKey pk1 = Box.PublicKey.fromBytes(decodeBytes(PK_1_B_64));
+    Box.PublicKey pk2 = Box.PublicKey.fromBytes(decodeBytes(PK_2_B_64));
+
+    ConcurrentNetworkNodes networkNodes =
+        new ConcurrentNetworkNodes(NodeUtils.url("127.0.0.1", firstOrionLauncher.nodePort()));
+    networkNodes.addNode(pk1, NodeUtils.url("127.0.0.1", firstOrionLauncher.nodePort()));
+    networkNodes.addNode(pk2, NodeUtils.url("127.0.0.1", secondOrionLauncher.nodePort()));
+
+    RequestBody partyInfoBody =
+        RequestBody.create(MediaType.parse(CBOR.httpHeaderValue), Serializer.serialize(CBOR, networkNodes));
+    OkHttpClient httpClient = new OkHttpClient();
+
+    final String firstNodeBaseUrl = NodeUtils.urlString("127.0.0.1", firstOrionLauncher.nodePort());
+    Request request = new Request.Builder().post(partyInfoBody).url(firstNodeBaseUrl + "/partyinfo").build();
+    await().atMost(5, TimeUnit.SECONDS).until(() -> getPartyInfoResponse(httpClient, request).nodeURLs().size() == 2);
+
+    memoryKeyStore = new MemoryKeyStore();
+  }
+
+  private ConcurrentNetworkNodes getPartyInfoResponse(OkHttpClient httpClient, Request request) throws Exception {
+    Response resp = httpClient.newCall(request).execute();
+    assertEquals(200, resp.code());
+    return Serializer.deserialize(HttpContentType.CBOR, ConcurrentNetworkNodes.class, resp.body().bytes());
+  }
+
+  @AfterEach
+  void tearDown() {
+    firstOrionLauncher.stop();
+    vertx.close();
+  }
+
+  @Test
+  void receiverCanViewWhenSentToPrivacyGroup() {
+    final EthClientStub firstClient = NodeUtils.client(firstOrionLauncher.clientPort(), firstHttpClient);
+    final EthClientStub firstNode = NodeUtils.client(firstOrionLauncher.nodePort(), firstHttpClient);
+
+    String[] addresses = new String[] {PK_1_B_64, PK_2_B_64};
+    final PrivacyGroup privacyGroup =
+        createPrivacyGroupTransaction(firstClient, addresses, PK_1_B_64, "testName", "testDescription");
+
+    EncryptedPayload payload = mockPayload();
+    var pushResult = push(firstNode, payload);
```

Comment:
remove `push` method from `NodeUtils` as it is not a "client" method. Use `firstClient.push(..)` instead